scapy/utils.py

```diff
@@ -1371,6 +1371,11 @@ def _write_header(self, pkt):
         if self.linktype is None:
             try:
                 self.linktype = conf.l2types[pkt.__class__]
+                # Import here to prevent import loops
+                from scapy.layers.inet import IP
+                from scapy.layers.inet6 import IPv6
+                if OPENBSD and isinstance(pkt, (IP, IPv6)):
```

Comment:
`tcpdump` on OpenBSD uses a specific DLT value for raw IP packets. This fixes some unit tests.